src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
@zhenlan you are correct you will not be able to ship a stable version of this package until all your dependencies are stable. Currently this package is "beta" so no worries there but you should be sure to track the beta dependencies closely if you plan to ship a stable package for SM.